client/app/widgets/Notifications/index.jsx

```diff
@@ -45,90 +54,85 @@ export class Notifications extends React.Component<Props, State> {
     const channel = pusher.subscribe(`private-${signedInKey}`);
```

Comment:
So the issue is that `signedInKey` can either be undefined or a number. If it's undefined, you can't do string interpolation of this value. So to bypass this type issue, I would just use `paramSignedInKey` directly (which should always be a number).